PWGHF/TableProducer/HFDsToKKPiCandidateSelector.cxx

```diff
@@ -0,0 +1,171 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file HFDsToKKPiCandidateSelector.cxx
+/// \brief Ds± → K± K∓ π± selection task
+///
+/// \author Stefano Politano <stefano.politano@cern.ch>, Politecnico and INFN Torino
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "PWGHF/DataModel/HFSecondaryVertex.h"
+#include "PWGHF/DataModel/HFCandidateSelectionTables.h"
+#include "Common/Core/TrackSelectorPID.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_prong3;
+using namespace o2::analysis::hf_cuts_ds_tokkpi;
+
+/// Struct for applying Ds to KKpi selection cuts
+struct HfDsTokkpiCandidateSelector {
+  Produces<aod::HFSelDsToKKPiCandidate> hfSelDsToKKPiCandidate;
+
+  Configurable<double> pTCandMin{"pTCandMin", 1., "Lower bound of candidate pT"};
+  Configurable<double> pTCandMax{"pTCandMax", 36., "Upper bound of candidate pT"};
+  // TPC
+  Configurable<bool> requireTPC{"requireTPC", true, "Flag to require a positive Number of found clusters in TPC"};
```

Comment:
`requireTPC` is not used

Reply:
Indeed, it is not needed inside the selector as the PID from TPC is already required in the trackselection step if I am not wrong (maybe @fgrosa can comment on this). However, the same line is present in the Dplus (https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/HFDplusToPiKPiCandidateSelector.cxx#L36) and Lc->K0sP (https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/HFLcK0sPCandidateSelector.cxx#L72) candidate selector and not used. Do you want me to remove this also from there?

Reply:
Ciao @stefanopolitano, indeed I would remove it from here: in the `HFTrackIndexSkimsCreator` the quality selection on tracks is applied, so there is no need to add other requirements here (or at least nTPCcls > 0 is not a requirement we would need anyway). You can remove it I would say, thanks!